src/rust/ide/src/model/suggestion_database.rs

```diff
@@ -98,6 +101,21 @@ impl Entry {
     pub fn with_name(self, name:impl Into<String>) -> Self {
         Self {name:name.into(),..self}
     }
+
+    /// Generates HTML documentation for documented suggestion
+    fn gen_doc(doc: Option<String>) -> Option<String> {
+        match doc {
+            Some(d) => {
+                let parser = DocParser::new_or_panic();
+                let output = parser.generate_html_doc_pure(d);
+                match output {
+                    Ok(result) => Some(result),
+                    Err(_)     => None,
+                }
+            },
+            None    => None,
```

Comment:
```suggestion
            None => None,
```
If lines are not consecutive, then we don't align anytihng.